src/js/tabs/receiver_msp.js

```diff
@@ -28,42 +33,54 @@ var
         Aux3: CHANNEL_MIN_VALUE,
         Aux4: CHANNEL_MIN_VALUE
     },
-    
+
     // First the vertical axis, then the horizontal:
     gimbals = [
         ["Throttle", "Yaw"],
         ["Pitch", "Roll"],
     ],
-    
+
     gimbalElems,
     sliderElems,
-    
+
     enableTX = false;
 
 // This is a hack to get the i18n var of the parent, but the localizePage not works
 const i18n = opener.i18n;
 
+let watchers = {
+	darkTheme: (val) => {
```

Comment:
You are using tabs here instead of 4 spaces - probably best change this in your editor. :space_invader:

Reply:
Sure, I'll commit this along with the dark theme for sliders.

Reply:
Yep. That's done. 
Also I restyle dark theme for sticks window. The first post was update accordingly.
Should I "Resolve conversion"?